### DISCUSSION TITLE
Execute Repeated commands

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#768] [Q&A] [answered]
I wanted to create a binding that executes an operation multiple times:
```
map } nextTab nextTab
```
This should jump to the next second tab

**Comment by gdh1995** [comment, marked as answer]
Most commands support "count prefix", so if `map v nextTab`, then `2v` means to go to the 2nd right tab. And you may also write `map } nextTab count=2`.

Vimium C doesn't remember what's your last command, so there's no command to do what this title says.
